### PR TITLE
Telegram Bot Boilerplate

### PR DESCRIPTION
**Pull request by riojohanes**
install grammy, grammyjs/menu, create bot.js to reply simple message

Fix this #1

### CODE REVIEW

**Comment by dimasriat** [comment]
don't include the node_modules.
solution: create `.gitignore` file and add `node_modules` inside it

**Comment by dimasriat** [comment]
almost correct, but the file changes still include the node_modules

image

need to include the .gitignore since the first commit for this branch.

clue: use `git reset --hard origin/main` but it will reset the progress, so you might need to back up first

**Comment by dimasriat** [comment]
Good, can you add .env to set the telegram api token? also don't forget to add the .env to .gitignore. And also create the .env.example to give the user the context of the .env

File: bot.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const dotenv = require("dotenv");
+const { Bot } = require("grammy");
+// const { Menu } = require("@grammyjs/menu");
+
+
+const TELEGRAM_BOT_API_TOKEN = process.env.TELEGRAM_BOT_API_TOKEN;
```
Comment (dimasriat) [inline bot.js:6]: just one more thing, add `dotenv.config();` before reading any env file

Comment (riojohanes) [reply bot.js:6]: ahh i forgot, thanks

**Comment by dimasriat** [comment]
LGTM!

**Review by dimasriat — APPROVED** [review]

**Pull request merged**